posthog/settings/web.py

```diff
@@ -417,6 +417,12 @@ def static_varies_origin(headers, path, url):
 PROXY_TARGET_CNAME = get_from_env("PROXY_TARGET_CNAME", "")
 PROXY_BASE_CNAME = get_from_env("PROXY_BASE_CNAME", "")
 
+# Cloudflare for SaaS proxy settings
+CLOUDFLARE_PROXY_ENABLED = get_from_env("CLOUDFLARE_PROXY_ENABLED", False, type_cast=str_to_bool)
+CLOUDFLARE_API_TOKEN = get_from_env("CLOUDFLARE_API_TOKEN", "")
+CLOUDFLARE_ZONE_ID = get_from_env("CLOUDFLARE_ZONE_ID", "")
```

Comment:
Have we decided on a hostname for this proxy service yet? I suspect yes since this is an existing feature. It would be really great if we could use a domain that isn't `posthog.com`, just to keep this programmatic DNS management separate from our most critical domain.

Reply:
I've used livehog.com for the initial API token. We can always switch it later if needed. The zone ID is `32562f8f172cf7e0cff6fbf2f543b831`.